pkg/storage/storage.go

```diff
@@ -787,6 +790,9 @@ func (s *storageModule) periodicallyCheckDiskShutdown() {
 // If a disk is on and it is not mounted then it is not supposed to be on, turn it off
 func (s *storageModule) shutdownDisks() {
 	for _, pool := range s.pools {
+		if _, mounted := pool.Mounted(); !mounted {
+			continue
```

Comment:
If the pool is not mounted, is there is a way to make sure that it's also shutdown? and shut it down if it's not.

Reply:
ah wait this should not be there indeed